srcpkgs/lcalc/files/test

```diff
@@ -0,0 +1,17 @@
+echo "### First 1000 zeros of zeta function"
```

Comment:
Maybe add a header to this file, explaining that its purpose is to verify both library and executable are working ok, and what the approach to developing the tests was.

Reply:
Done